test/manual/sort_by_proximity_test.cc

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include <chrono>
+
+#include <seastar/core/reactor.hh>
+#include <seastar/core/app-template.hh>
+#include <seastar/core/sstring.hh>
+#include <seastar/core/sleep.hh>
+#include <seastar/core/thread.hh>
+#include <seastar/rpc/rpc_types.hh>
+#include "inet_address_vectors.hh"
+#include "seastarx.hh"
+
+#include "locator/token_metadata.hh"
+
+#include "test/lib/log.hh"
+#include "test/lib/topology.hh"
+
+using namespace std::chrono_literals;
+using clock_type = std::chrono::high_resolution_clock;
+
+// Called in a seastar thread
+void test_sort_by_proximity(clock_type::duration duration) {
```

Comment:
probably we could leverage the facilities like `PERF_TEST_F()` provided by seastar? for instance, see https://github.com/scylladb/scylladb/blob/master/test/perf/perf_big_decimal.cc

Reply:
good idea, thanks

Reply:
In v3 ([bc3acd5](https://github.com/scylladb/scylladb/commit/bc3acd52e39c2f97e0a9ffc478e1a2c4db7f4051))